vendor/github.com/docker/go-connections/tlsconfig/certpool_go17.go

```diff
@@ -6,7 +6,7 @@ import (
 	"crypto/x509"
 	"runtime"
 
-	"github.com/Sirupsen/logrus"
```

Comment:
Did you change this line manually? The problem now is that the version linked in the version.json file is not in sync with this version anymore. This seems to become a bit trickier.